academic-services/knowledge-exploration-service/introduction.md

```diff
@@ -6,6 +6,9 @@ ms.date: 03/18/2018
 ---
 # About Microsoft Academic Knowledge Exploration Service
 
+> [!IMPORTANT]
+> We will be publishing a new version of MAKES early in the next calendar year (2020).  This version will be a turn-key solution to host MAKES in your Azure subscription.  For users of the Cognitive Service API, this will allow you to host your own un-throttled version of the API set provided by this service.
```

Comment:
"early in the next calendar year (2020)" => "in early 2020"
"For users of the Cognitive Service API, this will allow you to host your own un-throttled version of the API set provided by this service." =>"This new version will mirror the schema and API methods supported by [Project Academic Knowledge](https://www.microsoft.com/en-us/research/project/academic-knowledge/), enabling users to host their own un-throttled version of the API."

Reply:
Ok, will update.  Not sure how to respond your first topic, but was hoping to add this to the top level .yaml page, but doesn't look like that is supported.  Do you know of a way to do that?  I can certainly add this to the top of each page to catch folks that are coming in from search or other links, good suggestion.

Reply:
The only shared component is TOC.yml, however that is only for the table of contents. There might be a way to do this (insert common page content via resource link), however I'm not sure what it is.

Reply:
Oh, think I found something!
https://review.docs.microsoft.com/en-us/help/contribute/includes-best-practices?branch=master